core/src/main/scala/com/avast/clients/rabbitmq/rabbitmq.scala

```diff
@@ -37,4 +38,16 @@ package object rabbitmq {
     }
   }
 
+  private[rabbitmq] implicit class BasicPropertiesOps(val props: com.rabbitmq.client.AMQP.BasicProperties) extends AnyVal {
```

Comment:
Is there a reason why this method should be `private`?

Reply:
Well... since it's an extension method of raw AMQP Java properties which are not accessible with this client (they're wrapped into a case class), I see no point in keeping it public.